spec/eof.md

```diff
@@ -348,6 +381,31 @@ During scanning, for each instruction:
 
 Annotated examples of EOF formatted containers demonstrating several key features of EOF can be found in [this test file within the `evmone` project repository](https://github.com/ethereum/evmone/blob/master/test/unittests/eof_example_test.cpp).
 
+## Appendix: Creator Contract
+
+```solidity
+{
+/// Takes [index][salt][init_data] as input,
+/// creates contract and returns the address or failure otherwise
+/// init_data.length can be 0, but the first 2 words are mandatory
+let size := calldatasize()
+if lt(size, 64) { revert(0, 0) }
+let tx_initcode_index := calldataload(0)
+let salt := calldataload(32)
+let init_data_size := sub(size, 64)
+calldatacopy(0, 64, init_data_size)
+let ret := txcreate(tx_initcode_index, callvalue(), salt, 0, init_data_size)
+if iszero(ret) { revert(0, 0) }
+mstore(0, ret)
+return(0, 32)
+// Helper to compile this with existing Solidity (with --strict-assembly mode)
+function txcreate(a, b, c, d, e) -> f {
+    f := verbatim_5i_1o(hex"ed", a, b, c, d, e)
+}
+    
+}
+```
+
```

Comment:
Indentation?

Reply:
hm, we didn't originally have indentation here, not sure if that was intentional. I'll bring it back and also re-add whitespace which's gone missing for some reason, thx